browser/src/control/VRuler.ts

```diff
@@ -231,6 +232,19 @@ class VRuler {
 		return this._rWrapper;
 	}
 
+	_onRulerChanged() {
+		// update show ruler state on rulerChange event
+		this.options.showruler = this._map.uiManager.getBooleanDocTypePref(
```

Comment:
this method looks exactly like in the horizontal ruler -> could be shared
we need to remember about it when converting to TS -> I see no point doing base class in Leaflet style now...